docs/installation/server-config/config-reference.rst

```diff
@@ -211,6 +211,82 @@ bind to the default interface. For example:
     # Bind only to localhost.
     listeners=http://localhost:8088
 
+KSQL Processing Log Settings
+----------------------------
+
+These configurations control the behavior of the :ref:`KSQL processing log <ksql_processing_log>`.
+
+.. _ksql-processing-log-topic-auto-create:
+
+-------------------------------------
+ksql.processing.log.topic.auto.create
```

Comment:
we should be generating such docs _from_ the config... :(

(Though don't get me wrong - you adding the docs is great!)

Reply:
I looked at doing this at first and then punted on it. ConfigDef is set up to do this, but we need to do a few things:
- Integrate it into the sphinx build stuff, which is a fair bit of work - I think we'll need to add a build target that can generate the docs, then update the docs repo to use it for ksql and copy the artifact over to the ksql docs before running sphinx-build
- The format of the output is different from the rest of the config docs, so we'd want to do this for all the config docs. But we'd want to be sure to configure internal configs correctly in our code.

I think it makes sense to at least have docs first.

cc @JimGalasyn

Reply:
> I think it makes sense to at least have docs first.

Absolutely. Just raising this as its not something that's getting any attention. It's always a 'would it be nice if...'. 

Added https://github.com/confluentinc/ksql/issues/2423